# HG changeset patch
# User Rik <rik@octave.org>
# Date 1411879829 25200
#      Sat Sep 27 21:50:29 2014 -0700
# Node ID cd554c5b65ee2d5d0e3e55a5401b602deb73bdab
# Parent  6f3e922c51973bc27c3f210bf43c9c370bba94c5
Mark __clabel__ and __getlegenddata__ as internal functions not requiring BIST.

* __clabel__.m, __getlegenddata__.m: Add %!assert (1) to remove scripts from
requiring a BIST test.

diff --git a/scripts/plot/appearance/__clabel__.m b/scripts/plot/appearance/__clabel__.m
--- a/scripts/plot/appearance/__clabel__.m
+++ b/scripts/plot/appearance/__clabel__.m
@@ -122,8 +122,12 @@ function h = __clabel__ (c, v, hparent, 
         h = [h; ht];
       endif
     endfor
     i += clen+1;
   endwhile
 
 endfunction
 
+
+## No test needed for internal helper function.
+%!assert (1)
+
diff --git a/scripts/plot/appearance/__getlegenddata__.m b/scripts/plot/appearance/__getlegenddata__.m
--- a/scripts/plot/appearance/__getlegenddata__.m
+++ b/scripts/plot/appearance/__getlegenddata__.m
@@ -54,8 +54,12 @@ function [hplots, text_strings] = __getl
           text_strings(end+1) = dname;
         endif
       endif
     endif
   endfor
 
 endfunction
 
+
+## No test needed for internal helper function.
+%!assert (1)
+
